docs/resources/gaussdb_redis_instance.md

```diff
@@ -0,0 +1,180 @@
+---
+subcategory: "GaussDB"
+---
+
+# huaweicloud_gaussdb_redis_instance
+
+GaussDB for Redis instance management within HuaweiCoud.
+
+## Example Usage
+
+### create a gaussdb for redis instance with tags
+
+```hcl
+resource "huaweicloud_gaussdb_redis_instance" "test" {
+  name              = "gaussdb_redis_instance_1"
+  password          = var.password
+  flavor            = "geminidb.redis.xlarge.4"
+  volume_size       = 100
+  vpc_id            = var.vpc_id
+  subnet_id         = var.subnet_id
+  security_group_id = var.secgroup_id
+  availability_zone = var.availability_zone
+
+  tags = {
+    foo = "bar"
+    key = "value"
+  }
+}
+```
+
+### create a gaussdb redis instance with backup strategy
+
+```hcl
+resource "huaweicloud_gaussdb_redis_instance" "test" {
+  name              = "gaussdb_redis_instance_1"
+  password          = var.password
+  flavor            = "geminidb.redis.xlarge.4"
+  volume_size       = 100
+  vpc_id            = var.vpc_id
+  subnet_id         = var.subnet_id
+  security_group_id = var.secgroup_id
+  availability_zone = var.availability_zone
+
+  backup_strategy {
+    start_time = "03:00-04:00"
+    keep_days  = 14
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the Redis instance resource.
+  See [Region and Endpoints](https://developer.huaweicloud.com/intl/en-us/endpoint?GaussDB%20NoSQL) for more detail.
+  If omitted, the provider-level region will be used. Changing this creates a new Redis instance resource.
+
+* `availability_zone` - (Required, String, ForceNew) Specifies the AZ name.
+  See [Region and Endpoints](https://developer.huaweicloud.com/intl/en-us/endpoint?GaussDB%20NoSQL) for more detail.
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the instance name, which can be the same
+  as an existing instance name. The value must be 4 to 64 characters in
+  length and start with a letter. It is case-sensitive and can contain
+  only letters, digits, hyphens (-), and underscores (_).
+  Chinese characters must be in UTF-8 or Unicode format.
+
+* `flavor` - (Required, String) Specifies the instance specifications. For details, 
+  see [DB Instance Specifications](https://support.huaweicloud.com/intl/en-us/redisug-nosql/nosql_05_0059.html).
+  Do nothing in update method if change this parameter.
+
+* `node_num` - (Optional, Int) Specifies the number of nodes, ranges from 2 to 12. Defaults to 2.
```

Comment:
actually, the default value is **3** in the schema, please check it.

Reply:
fixed